pom.xml

```diff
@@ -2,7 +2,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>edu.harvard.huit.lts</groupId>
 	<artifactId>fits-service</artifactId>
-	<version>1.3.0-SNAPSHOT</version>
+	<version>1.6.0-SNAPSHOT</version>
```

Comment:
Do the versions need to match? What happens if you need to do a security fix for the servlet? Will there be a corresponding FITS release too?

Reply:
They don't need to match. This was an arbitrary decision in scrum this morning since previous version, 1.4.0, 1.4.1, 1.5.5, matched FITS versions. So we didn't want to go to an earlier version. Ask Chris if you have more questions.

Reply:
Adding to what Dave wrote, `FITSservlet` has its own version (was `1.3.0-SNAPSHOT`), but the docker images were tagged for the FITS version (last: `1.5.5`). This ended up being confusing and we wanted to use the `FITSservlet` version in a meaningful way. If we were to start with `FITSservlet:1.3.0-SNAPSHOT`, we would eventually have collisions with existing tags, so we want to bump the version sufficiently to avoid that. This is also leading to some confusion, as it appears to be pegged to FITS, which it is not. @daveneiman, perhaps it's time to increment FITSservlet to `2.0.0` to avoid any confusion?

Reply:
Oops! Sorry, I forgot that part of our conversation this morning.

Reply:
POM version bumped to 2.0.0-SNAPSHOT